lib/scala/project-manager/src/main/scala/org/enso/projectmanager/boot/Cli.scala

```diff
@@ -45,6 +46,13 @@ object Cli {
       .desc("Checks the version of the Enso executable.")
       .build()
 
+    val native: cli.Option = cli.Option.builder
+      .longOpt(NATIVE_OPTION)
+      .desc(
+        "(experimental) Attempts to use the native-image of any subprocess."
```

Comment:
Can't we just use the `bin/enso` or `bin/enso.exe` if it is generated without any (even experimental) option? 

Btw. there already is support for `--jvm` option. We should align with it. Maybe just expose `--jvm` as an option to `project-manager` and just pass it on to `bin/enso`? 

Anyway the same can already be done with `ENSO_OPTS` environment variable and it may be enough for development purposes.